cmp/mask/DisabledMask.js

```diff
@@ -0,0 +1,49 @@
+/*
+ * This file belongs to Hoist, an application development toolkit
+ * developed by Extremely Heavy Industries (www.xh.io | info@xh.io)
+ *
+ * Copyright © 2018 Extremely Heavy Industries Inc.
+ */
+
+import {Component} from 'react';
+import {hoistComponent, elemFactory} from 'hoist/core';
+import {frame} from 'hoist/layout';
+import {overlay} from 'hoist/kit/blueprint';
+
+import './Mask.scss';
+
+/**
+ * Simple Mask for Disabled or Inactive components.
+ */
+@hoistComponent()
+export class DisabledMask extends Component {
+
+    BACKGROUND = 'rgba(0,0,0, 0.60)';
```

Comment:
Quite a bit darker than the loadMask. I found that, without the spinner, 25% wasn't enough to clearly tell me the component wasn't active. I have a demo branch in PIQ-React with a button to temporarily disable the positions grid:

https://github.com/exhi/piq-react/tree/demoDisabledMask